ts/Series/RAFDeferDataLabels.ts

```diff
@@ -0,0 +1,87 @@
+/* *
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+import type NetworkgraphSeries from './Networkgraph/NetworkgraphSeries';
+import type PackedBubbleSeries from './PackedBubble/PackedBubbleSeries';
+import type SVGElement from '../Core/Renderer/SVG/SVGElement';
+
+import U from '../Core/Utilities.js';
+const { syncTimeout } = U;
+
+import A from '../Core/Animation/AnimationUtilities.js';
+const { animObject } = A;
+
+type RAFSeries = (NetworkgraphSeries | PackedBubbleSeries);
```

Comment:
What is RAF? 🤔

Reply:
Request Animation Frame. I tried to find a name specific to simulated series, but struggled to find anything better.

Reply:
SimulationSeries? These series run a physical simulation instead of plain animation.

Reply:
Okay, I'll rename the type to `type SimulationSeries = (NetworkgraphSeries | PackedBubbleSeries)`.

Should the name of the file stay the same? Or should it also include _SimulationSeries_?

Reply:
It should also include SimulationSeries. I think `SimulationSeriesUtilities.ts` would be a descriptive name.